pkgs/by-name/ru/rustdesk-flutter/package.nix

```
@@ -162,6 +163,8 @@ flutter316.buildFlutterApplication rec {
   patches = [ ./make-build-reproducible.patch ];
 
   postPatch = ''
+    substituteInPlace $cargoDepsCopy/libappindicator-sys-*/src/lib.rs \
+      --replace-fail "libayatana-appindicator3.so.1" "${libayatana-appindicator}/lib/libayatana-appindicator3.so.1"
```

In case the `outputs` of `libayatana-appindicator` get changed

```suggestion
      --replace-fail "libayatana-appindicator3.so.1" "${lib.getLib libayatana-appindicator}/libayatana-appindicator3.so.1"
```

---

Please try to give as more comments as possible in one review.